invenio_app_ils/ui/src/invenio_app_ils/common/config/invenioConfig.js

```diff
@@ -1,7 +1,200 @@
+            order: 2,
+            title: 'Best match',
+          },
+          {
+            default_order: 'asc',
+            field: 'available_items',
+            order: 3,
+            title: 'Available Items',
+          },
+          {
+            default_order: 'desc',
+            field: 'mostloaned',
+            order: 4,
+            title: 'Most loaned',
+          },
+        ],
+      },
+      sortOrder: ['asc', 'desc'],
+    },
+  },
+  editor: {
+    url: '/editor',
+  },
+  eitems: {
+    search: {
+      aggs: [],
+      sortBy: {
+        onEmptyQuery: 'mostrecent',
+        values: [
+          {
+            field: 'mostrecent',
+            order: 1,
+            title: 'Newest',
+          },
+          {
+            field: 'bestmatch',
+            order: 2,
+            title: 'Best match',
+          },
+        ],
+      },
+      sortOrder: ['asc', 'desc'],
+    },
+  },
+  items: {
+    canCirculateStates: ['CAN_CIRCULATE'],
+    search: {
+      aggs: ['status', 'medium', 'circulation'],
+      sortBy: {
+        onEmptyQuery: 'mostrecent',
+        values: [
+          {
+            field: 'mostrecent',
+            order: 1,
+            title: 'Newest',
+          },
+          {
+            field: 'bestmatch',
+            order: 2,
+            title: 'Best match',
+          },
+        ],
+      },
+      sortOrder: ['asc', 'desc'],
+    },
+  },
+  loans: {
+    search: {
+      aggs: ['state'],
+      sortBy: {
+        onEmptyQuery: 'mostrecent',
+        values: [
+          {
+            field: 'mostrecent',
+            order: 1,
+            title: 'Newest',
+          },
+          {
+            field: 'bestmatch',
+            order: 2,
+            title: 'Best match',
+          },
+        ],
+      },
+      sortOrder: ['asc', 'desc'],
+    },
+  },
+  max_results_window: 10000,
+  patrons: {
+    search: {
+      aggs: [],
+      sortBy: {
+        onEmptyQuery: null,
+        values: [],
+      },
+      sortOrder: ['asc', 'desc'],
+    },
+  },
+  relationTypes: [
+    {
+      id: 3,
+      label: 'Multipart Monograph',
+      name: 'multipart_monograph',
+    },
+    {
+      id: 4,
+      label: 'Serial',
+      name: 'serial',
+    },
+    {
+      id: 0,
+      label: 'Language',
+      name: 'language',
+    },
+    {
+      id: 1,
+      label: 'Edition',
+      name: 'edition',
+    },
+    {
+      id: 2,
+      label: 'Other',
+      name: 'other',
+    },
+  ],
+  rest_mimetype_query_arg_name: 'format',
+  series: {
+    search: {
+      aggs: ['moi'],
+      sortBy: {
+        onEmptyQuery: 'mostrecent',
+        values: [
+          {
+            field: 'mostrecent',
+            order: 1,
+            title: 'Newest',
+          },
+          {
+            field: 'bestmatch',
+            order: 2,
+            title: 'Best match',
+          },
+        ],
+      },
+      sortOrder: ['asc', 'desc'],
+    },
+  },
+  support_email: 'info@inveniosoftware.org',
```

Comment:
this `support_email` might be something you will want to inject from the env or a js when deploying in a specific env. Did you figure out a way to do that?